### ISSUE TITLE
i18n content with multiple_folders is broken in DecapCMS 3.0.0

### ISSUE THREAD

**Issue by Billybobbonnet** [decaporg/decap-cms#6913]
While it works properly with the versions prior to 3.0.0, it fails to display content after 3.0.0

Here is my `  config.yml`
```
backend:
  name: github
  repo: XXX/seed
  branch: main
  base_url: https://XXXXXXXXi.execute-api.eu-west-2.amazonaws.com/
  auth_endpoint: "/dev/auth"
  cms_label_prefix: "XXX/"
publish_mode: editorial_workflow
media_folder: "public"
public_folder: "/"
site_url: https://XXX.fr
logo_url: /icon.png
show_preview_links: false
slug:
  encoding: "ascii"
  clean_accents: true
  sanitize_replacement: "-"
i18n:
  structure: multiple_folders
  locales: [en, fr]
  default_locale: en
collections:
  - name: "editorial" 
    label: "Editorial content" # Used in the UI
    i18n: true
    slug: "{{id}}"
    folder: "content/pages" # The path to the folder where the documents are stored
    create: true # Allow users to create new documents in this collection
    fields: 
      - { label: id, name: id, widget: string, i18n: duplicate }
      - { label: Body, name: body, widget: markdown, i18n: true }
```

These are very simple documents. Everything works when I simply change the script to import from 
`https://unpkg.com/decap-cms@^3.0.0/dist/decap-cms.js` to `https://unpkg.com/netlify-cms@^2.10.17/dist/netlify-cms.js`

**Comment by Billybobbonnet** [comment]
Note that it fails to load the list of the i18n collection items, but if I target the URL of an item, it loads the page properly. I investigated if it was related to the slug not being "title" but it didn't help.

**Comment by juniyadi-pintar** [comment]
Thanks @Billybobbonnet 

I have same issue, when using multiple_folders, content not load and showing "No Entries"

But after switch to https://unpkg.com/netlify-cms@^2.10.17/dist/netlify-cms.js , it's working fine

**Comment by demshy** [comment]
Hey guys, thanks for your patience and great issue details. This should be fixed in 3.0.9, would you mind trying it out?

**Comment by juniyadi-pintar** [comment]
Hi @demshy , i can confirm. this is working now using 3.0.9
Thanks

<img width="707" alt="Screenshot 2023-10-12 at 10 48 04" src="https://github.com/decaporg/decap-cms/assets/117881249/6bdd0888-3dba-4f48-b694-97f689c5439c">
